bundles/core/org.eclipse.smarthome.core.thing.xml/thing-description-1.0.0.xsd

```diff
@@ -132,4 +133,18 @@
         </xs:simpleContent>
     </xs:complexType>
 
-</xs:schema>
+	<xs:complexType name="properties">
+		<xs:sequence>
+			<xs:element name="property" type="thing-description:property" minOccurs="1" maxOccurs="unbounded" />
+		</xs:sequence>
+	</xs:complexType>
+
+	<xs:complexType name="property">
+		<xs:simpleContent>
+			<xs:extension base="xs:string">
+				<xs:attribute name="name" type="xs:string" use="required" />

```

Comment:
Well, for me it is OK. Just looked at how Java Property files restrict the key, but also there everything is allowed. So I don´t see a technical reason, why the key should be restricted, but maybe we could advice to use camel case like "modelId". @kaikreuzer any concerns?